Libraries/Opc.Ua.Client/Subscription.cs

```diff
@@ -1876,71 +1963,32 @@ private void AdjustCounts(ref uint keepAliveCount, ref uint lifetimeCount)
         /// <summary>
         /// Processes the incoming messages.
         /// </summary>
-        private async Task OnMessageReceived()
+        private void OnMessageReceived()
```

Comment:
Because now there is only a single worker task per subscription, the semaphoreslim is not needed anymore to gate the Tasks. Semaphoreslim was the only reason to make it async. Instead now the worker is async and supposedly task friendly.